### DISCUSSION TITLE
I can't access the wordpress database

### DISCUSSION THREAD

**Discussion by oznisosa** [EasyEngine/easyengine#1527] [Support Requests]
Hello good Morning.

First of all I thank you for everything.

Please I need help; I can't access the wordpress database, I need to access the database to manually drop tables.

These are the steps that I followed.

1. and executed the following code.   =    ee admin-tools enable example.com
2. and executed the following comment to see the access data. =   ee auth list global

and I tried to access through this url, I did it by copying and pasting the access data but nothing happens.
example.com/ee-admin/

Also and tried directly to mysql with this url  = example.com/ee-admin/pma    ; with the data obtained from the following command.

But no matter how hard I try, I can't access.

My wordpress website is new, I did it with easyengine. this is the link arteanc.com

in followed the guide of the following link:   https://easyengine.io/handbook/admin-tools/

Please hope someone can help me.

Maybe there is something you are doing wrong.

Thanks a million in advance.

best regards.

**Comment by mrrobot47** [comment]
If you want CLI access to the database, run the following commands:

```bash
ee shell arteanc.com
wp db cli
```

Not sure why admin tools is not working for you, can you please share what happens when you open `/ee-admin`? Do you get 404 or some redirect?
